### PR TITLE
Rebuild for aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #46 

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -3,5 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: default, linux_ppc64le: default}
```
Comment (hmaarrfk) [inline conda-forge.yml]: Should we try it with `native`?
https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L1329
```suggestion
provider: {linux_aarch64: native, linux_ppc64le: default}
```

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml (hunk 1 above)
Comment (xhochy) [reply conda-forge.yml]: Wasn't aware of that!

Comment (hmaarrfk) [reply conda-forge.yml]: I think the time limit is real tight. But this was some cool work done by @jmreicha and isuruf

**Comment by hmaarrfk** [comment]
Are we hitting this bug?
https://github.com/ipython/ipykernel/pull/412

**Comment by hmaarrfk** [comment]
@conda-forge/arm-arch can somebody with superpowers activate 
https://cloud.drone.io/conda-forge/ipykernel-feedstock
please?

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
@xhochy  I think marius just enabled drone here. wanna add some whitespace to give it a try again? Maybe add back matplotlib but with selectors skipping aarch64 and ppc64le

**Comment by xhochy** [comment]
@hmaarrfk I have this in my `~/.gitconfig`:

```
[alias]
  fix = !git commit --amend --no-edit && git push --force
```

Triggers a CI rebuild without adding a commit or sending endless mails to reviewer (Github doesn't send mails on force-pushes without new commits).

**Comment by hmaarrfk** [comment]
Hmm. Ok. I just can't push to your branch. Sorry :(.

Seems like it was a bug with emulation????? Or a race condition exarcebated by qemu.

**Comment by xhochy** [comment]
@conda-forge/arm-arch @conda-forge/ipykernel This is ready to be merged.

**Comment by bollwyvl** [comment]
Hooray!

**Pull request merged**